modules/virt-preparing-container-disk-for-vms.adoc

```diff
@@ -7,6 +7,16 @@
 
 You must build a container disk with a virtual machine image and push it to a container registry before it can used with a virtual machine. You can then either import the container disk into a PVC using a data volume and attach it to a virtual machine, or you can attach the container disk directly to a virtual machine as an ephemeral `containerDisk` volume.
 
+[NOTE]
+====
+The size of a disk image inside a container disk is limited by the maximum layer size of the registry where the container disk is hosted. For Red Hat Quay, the default limit is 20Gi. You can configure the `MAXIMUM_LAYER_SIZE` field in the Red Hat Quay configuration schema. For more details, see the schema for Red Hat Quay configuration in the link:https://access.redhat.com/documentation/en-us/red_hat_quay/[Manage Red Hat Quay] documentation.
+====
+
+[IMPORTANT]
+====
+Using large container disks may cause an increase in I/O traffic that impacts worker nodes. In some cases, this can lead to unavailable nodes. See link:https://access.redhat.com/solutions/3396081[OpenShift Container Platform nodes are experiencing disk pressure] for resolution.
```

Comment:
s/may/might

...but I have some thoughts on this :) 

We shouldn't link to KCS articles unless we really have to, for quality control and because you have to log in to view them. This particular KCS also references OCP 3.x; even if the content is up-to-date, this might make a customer think they're in the wrong place. 

Looking at the KCS itself, the resolution is: "First, please ensure garbage collection and pruning are enabled for the cluster. Also enable log rotation of docker and other logs."

I searched for garbage collection and pruning in the OCP docs and found the following docs that you can xref to:

* [Pruning objects](https://docs.openshift.com/container-platform/4.9/applications/pruning-objects.html)
* [Garbage collection](https://docs.openshift.com/container-platform/4.9/nodes/nodes/nodes-nodes-garbage-collection.html)

The log rotation part could refer to numerous types of logs, but the KCS mentions issues with Docker logs specifically. Maybe we could say:
`Configure log rotation if link:https://docs.docker.com/config/containers/logging/configure/[Docker] or other logs consume too much disk space.`

So the final draft might look something like this (plus xrefs):

```
[IMPORTANT]
====
If you use large container disks, I/O traffic might increase, impacting worker nodes. This can lead to unavailable nodes. You can resolve this by:

* Pruning objects. (xref to https://docs.openshift.com/container-platform/4.9/applications/pruning-objects.html)
* Configuring garbage collection. (xref to https://docs.openshift.com/container-platform/4.9/nodes/nodes/nodes-nodes-garbage-collection.html)
* Configuring log rotation, if link:https://docs.docker.com/config/containers/logging/configure/[Docker] or other logs consume too much disk space.
====
```

Reply:
Yes, this sounds like a better approach to me. I was told that if the solution is verified, then it's linkable but that might not be the best thing to do especially in this case for the reasons you mention.

@fabiand, let me know what you prefer to do. Thanks Pan!

Reply:
Hm. I think the kcs might have changed since I initially looked at it - nvm.

I'd keep the first two links, but not the docker one (not applicable in OCP)

Reply:
@bgaydosrh It just occurred to me that this is a module, so we cannot (currently) xref in it. Can you move this block to the assembly now that it's going to contain xrefs?

Reply:
Great catch @ousleyp - Yes, I'll move that to the assembly.

Reply:
Moved the [Important] block to the assembly as suggested.